packages/launcher/src/renderer/launcher/settings/OwnAppDetailView.js

```diff
@@ -241,7 +242,9 @@ export class OwnAppDetailView extends Component<Props, State> {
     })
   }
 
-  onPressSubmitFoReview = () => {}
+  onPressSubmitFoReview = () => {
+    shell.openExternal('https://mainframehq.github.io/mainframe-os/docs/mft')
```

Comment:
@sarahwiley I think we should use the canonical address (`https://docs.mainframe.com/...`) for this. I'm not sure how the redirections will behave otherwise.

Reply:
Updated.